doc/schema.md

```diff
@@ -0,0 +1,190 @@
+# Parquet (and Avro) Schema for FHIR Resources
+
+In this document we review the mapping (a.k.a. projection) from FHIR resources
+to Parquet schema. A few high-level reminders:
+
+- The basic idea for the projection follows the
+  [Simplified SQL Projection of FHIR Resources](https://github.com/FHIR/sql-on-fhir/blob/master/sql-on-fhir.md)
+  proposal.
+- The projection is done using a forked version of the
+  [Bunsen library](https://github.com/google/fhir-data-pipes/tree/master/bunsen).
+  The entry point for this conversion logic is
+  [AvroConverter](https://github.com/google/fhir-data-pipes/blob/master/bunsen/bunsen-avro/src/main/java/com/cerner/bunsen/avro/AvroConverter.java).
+  As the name suggests, the conversion logic is from FHIR
+  [StructureDefinition](https://hl7.org/fhir/structuredefinition.html)
+  to [Apache Avro](https://avro.apache.org/docs/1.11.1/specification/).
+- Conversion from Avro to Parquet is done using the
+  [parquet-avro](https://www.javadoc.io/static/org.apache.parquet/parquet-avro/1.14.0/org/apache/parquet/avro/package-summary.html#package.description)
+  library.
+
+If you want to look at examples before reading the details, you can check
+[Patient_no-extension.schema](doc/Patient_no-extension.schema) for the
+projection of base [Patient](https://hl7.org/fhir/patient.html) resource.
+[Patient_US-Core.schema](Patient_US-Core.schema) provides an example for
+[US Core Patient profile](https://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-patient.html).
+To see the intermediate Avro schema for this resource, see
+[us-core-patient-schema.json](https://github.com/google/fhir-data-pipes/blob/master/bunsen/bunsen-avro/src/test/resources/r4-us-core-schemas/us-core-patient-schema.json).
+
+## Type mapping rules
+
```

Comment:
Can we also specify the Fhir versions that are currently supported by us i.e. R4 and Stu3.

Reply:
Yes, you are right, the mapping rules are generic. The reason I was asking to add is because the application currently supports R4 and Stu3 and hence I thought it could be added here. But I think what you said also makes sense.